pip_faster.py

```diff
@@ -381,7 +401,7 @@ def prepare_files(self, finder, **kwargs):
         for req in wb.build():
             link = optimistic_wheel_search(req, finder.find_links)
             if link is None:
-                logger.notify('\033[1;91m[SLOW!]\033[0m no wheel found for %s after building (couldn\'t be wheeled?)', req)
+                logger.error('SLOW!! no wheel found after building (couldn\'t be wheeled?): %s', req)
```

Comment:
The red ANSI escapes are reimplemented via `logger.error`; pip configures it to be red.